.github/workflows/bindings-python.yml

```diff
@@ -67,18 +67,57 @@ jobs:
         working-directory: bindings/python
         run: tox -e format
 
+  lint:
```

Comment:
Should this also `needs: format` ?

Reply:
Would delay it a few seconds, but I don't really care if we make it dependent or not